src/scripts/tasks.coffee

```diff
@@ -81,3 +83,37 @@ module.exports = (robot) ->
       msg.send "#{displays[...].join("\n\n")}"
     .catch (error) ->
       msg.reply error
+
+  robot.respond /octo deploy (.+) > ([0-9a-zA-Z.-]+) > (.+)/i, (msg) ->
+    params =
+      project: msg.match[1]
+      version: msg.match[2]
+      env: msg.match[3]
+    if !params.project || !params.version || !params.env
+      msg.reply 'can\'t deploy if you don\'t specify all the params'
```

Comment:
Doesnt Seem as helpful, maybe output something more specific for each param or just list the params.